src/styles/main.scss

```
@@ -24,3 +25,10 @@
 .tna-\!--align-centre {
   text-align: center !important;
 }
+
+.development-notice {
+  padding: spacing.space(1);
+
+  border: 0.2em #00b0ff solid;
```

Would this work for you?

```suggestion
  @include colour.blue-accent;
  @include colour.thick-keyline-accent;
```